dotnet/docs/release-notes.mdx

```diff
@@ -323,7 +323,7 @@ await Expect(Page.GetByPlaceholder("Search docs")).ToHaveValueAsync("locator");
 ```
 
 ### New APIs
-- Option `reason` in [Page.CloseAsync()](/api/class-page.mdx#page-close), [BrowserContext.CloseAsync()](/api/class-browsercontext.mdx#browser-context-close) and [Browser.CloseAsync()](/api/class-browser.mdx#browser-close). Close reason is reported for all operations interrupted by the closure.
+- Options `reason` in [Page.CloseAsync()](/api/class-page.mdx#page-close), `reason` in [BrowserContext.CloseAsync()](/api/class-browsercontext.mdx#browser-context-close) and `reason` in [Browser.CloseAsync()](/api/class-browser.mdx#browser-close). Close reason is reported for all operations interrupted by the closure.
```

Comment:
Ideally we make it `Reason`

Reply:
This will happen in https://github.com/microsoft/playwright.dev/pull/1531.